core/chains/evm/builder/builder.go

```diff
@@ -0,0 +1,49 @@
+package builder
+
+import (
+	"github.com/smartcontractkit/sqlx"
+
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/forwarders"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/txmgr"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+)
+
+func BuildNewTxm(
```

Comment:
Does the `Build` prefix add value? We are already in `package builder`.
```suggestion
func NewTxm(
```